apps/src/templates/progress/progressStyles.js

```diff
@@ -64,6 +77,215 @@ export const hoverStyle = {
     'background-color .2s ease-out, border-color .2s ease-out, color .2s ease-out'
 };
 
+/**
+ * ======================================
+ * Bubble styles
+ * ======================================
+ */
+
+export const BubbleSize = makeEnum('dot', 'letter', 'full');
+export const BubbleShape = makeEnum('circle', 'diamond', 'pill');
+
+/**
+ * Note: these constants will be removed in favor of `bubbleSizes` below once
+ * we finish cleaning up all of our bubble components.
+ */
+export const DOT_SIZE = 30;
+export const DIAMOND_DOT_SIZE = 22;
+export const SMALL_DOT_SIZE = 9;
+export const SMALL_DIAMOND_SIZE = 6;
+
+const LARGE_FONT = 16;
+const SMALL_FONT = 12;
+
+const bubbleSizes = {
```

Comment:
Maybe add a comment here that indicates that these are sizes using border-box sizing?

Reply:
my thinking here was that the size of the bubble should be the size of the bubble, i.e. if an element has a border and we want to know the size of the element, that element _should_ be using `border-box`. in other words, imo the old version needed a comment because it was using `content-box` while needing to account for the border, but the new version doesn't. that was part of the "cleanup" here. but if you think this is confusing without a comment i will add one.

Reply:
Since there are different ways to interpret size, I thought it might be helpful to indicate which "size" these values refer to. Up to you. (Also ok to defer to a later PR since we're doing more work here soon.)